yodalib/decompilers/ghidra/interface.py

```diff
@@ -270,6 +270,44 @@ def _structs(self) -> Dict[str, Struct]:
             name: Struct(name, size, members=self._struct_members_from_gstruct(name)) for name, size in name_sizes
         } if name_sizes else {}
 
+    @ghidra_transaction
+    def _set_enum(self, enum: Enum, **kwargs) -> bool:
+        old_ghidra_enum = self.ghidra.currentProgram.getDataTypeManager().getDataType(enum.name)
+        data_manager = self.ghidra.currentProgram.getDataTypeManager()
+        handler = self.ghidra.import_module_object("ghidra.program.model.data", "DataTypeConflictHandler")
+        enumType = self.ghidra.import_module_object("ghidra.program.model.data", "EnumDataType")
+        categoryPath = self.ghidra.import_module_object("ghidra.program.model.data", "CategoryPath")
+        parts = enum.name.split('/')
```

Comment:
@Flipout50 this logic seems incorrect to me. So you are saying that a user in BinSync the lifting platform, needs to name their Enum starting with a `/`? That's not how it looks in C or in any other decompiler. Can we make it more generic here so they can name it normal and then you fix it up when setting it in Ghidra? Additionally, when we get Enums, let's not have the `/` in them. Only Ghidra should know about the `/`.